presto-docs/src/main/sphinx/installation/spark.rst

```diff
@@ -0,0 +1,69 @@
+=========================
+Executing Presto on Spark
+=========================
+
+When there is a need to execute long running queries, Presto offers
+an option to scale the resource management using Spark, as it doesn'f offer
+one currently by itself. This way Presto can be executed as a stand alone 
+application on Spark.
+
+Steps
+-----
+
+Download the Presto Spark package tarball, :maven_download:`spark-package` 
+and the Presto Spark laucher, :maven_download:`spark-launcher`. Keep both the
+files at, say, *example* directory. We assume here a two node Spark cluster
+with four cores each, thus giving us total eight cores.
+
+The following is an example ``config.properties``:
+
+.. code-block:: none
+
+    task.concurrency=4
+    task.max-worker-threads=4
+    task.writer-count=4
+     
+The details about properties are available at :doc:`/admin/properties`.
+Note that ``task.concurrency``, ``task.writer-count`` and 
```

Comment:
@arhimondr : I remember you mentioned the number of threads should be slightly higher than the number of physical cores to get the best resource utilization?

Reply:
Incorporated changes by @wenleix ..Please review, thanks